snoop/html.py

```diff
@@ -0,0 +1,107 @@
+import re
+from bs4 import BeautifulSoup, UnicodeDammit
+import lxml.html
+import lxml.html.clean
+import lxml.etree
+
```

Comment:
I'd rather not have two blank lines. I know pep-8 recommends them. But i think they cause you to see less of the code at one time.